packages/layout/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@react-spectre/layout",
+  "version": "1.0.0-6",
+  "description": "React components for Spectre.css's flexbox grid system.",
+  "jsnext:main": "src/index.js",
+  "main": "dist/layout.js",
+  "browser": "dist/layout.umd.js",
+  "module": "dist/layout.es.js",
+  "repository": "https://github.com/react-spectre/react-spectre/tree/master/packages/layout",
+  "author": "Ángel Piña <six.aps@gmail.com>",
```

Comment:
If this doesn't give me issues with NPM orgs, scoped packages or anything else than I will let your authorship.

Reply:
no worries men, just let me know and I'll change it.